server/src/scheduler/component/plugin-trigger-scheduler-component.js

```diff
@@ -1,3 +1,43 @@
+const collectAllPlugins = pluginInstancesByPlugin => {
+    return Object.keys(pluginInstancesByPlugin).map(pluginKey => {
+        const plugin = pluginInstancesByPlugin[pluginKey]
+        return Object.keys(plugin).map(pluginInstanceKey => {
+            const pluginInstance = plugin[pluginInstanceKey]
+            return {
+                pluginInstanceId: pluginInstanceKey,
+                configuration: pluginInstance.configuration,
+                queryUrl: pluginInstance.query_url
+            }
+        })
+    }).reduce(
+        (accumulator, current) => accumulator.concat(current),
+        []
+    )
+}
+
+const tickAllPlugins = pluginInstances => {
+    console.log(pluginInstances)
+    const queryAllPlugins = pluginInstances.map(instance => {
+        const requestPayload = {
+            pluginInstanceId: instance.pluginInstanceId,
+            configuration: instance.configuration,
+            callbackUrl: `https://us-central1-${projectId}.cloudfunctions.net/pluginCallback?pluginInstanceId=${instance.pluginInstanceId}`,
```

Comment:
this `projectId` needs to come from https://github.com/novoda/dashboards/pull/40/files#diff-da93c66952adaf756f0bc23000673d31R41